rotkehlchen/tests/unit/decoders/test_paraswap.py

```diff
@@ -17,6 +17,7 @@
     A_KP3R,
     A_LINK,
     A_POLYGON_POS_MATIC,
+    A_POLYGON_POS_USDC,
```

Comment:
If it is only used for tests don't move it since the assets on this file are special